actions1/cla_check.js

```diff
@@ -0,0 +1,96 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview File to check if PR Author has signed the CLA
+ */
+
+const core = require('@actions/core');
+const { context } = require('@actions/github');
+const { execSync } = require('child_process');
+const { google } = require('googleapis');
+
+const SHEETS_TOKEN = process.env.SHEETS_TOKEN;
+const CREDENTIALS = JSON.parse(process.env.SHEETS_CRED);
+const SPREADSHEET_ID = '1naQC7iEfnro5iOjTFEn7iPCxNMPaPa4YnIddjT5CTM8';
```

Comment:
Should we set this through the env variables too?

Reply:
Yes, That would be a good idea. DONE